crux-test/test/crux/fixtures/calcite.clj

```diff
@@ -0,0 +1,26 @@
+(ns crux.fixtures.calcite
```

Comment:
given this is a new fixture, worth putting it in `crux-test/src` instead, so that it can be used from tests in the individual modules?

Reply:
I'd like to do this in a subsequent PR - which would also involve moving the calcite test into the calcite/sql module and making full use of test/src. I want to properly scratch this itch in one go.